engine/src/main/java/org/terasology/world/generation/WorldFacet2D.java

```diff
@@ -24,10 +24,10 @@
     /**
      * @return The region of the world covered by this facet
      */
-    Rect2i getWorldRegion();
+    BlockAreac getWorldRegion();
```

Comment:
I think we should rename this to `getWorldArea`. I would like to reserve `region` for 3D block regions, and `area` for 2d block areas.
```suggestion
    BlockAreac getWorldArea();
```